code/API_definitions/Discovery/simple_edge_discovery.yaml

```diff
@@ -1,392 +1,444 @@
+    
+    Once you have the name of the closest MEC platform to the user device, you may:
+    
+    * connect your application client to your server instance on that MEC platform. Note: the address of that server instance is not part of the API response, but should be known in advance.
+    * or, if you have no instance on that MEC platform, you may wish to deploy one there.
+    
+     # Usage
+     
+     The API may be called either by an application client hosted on a device attached to the operator network (i.e. phone, tablet), or by a server.
+     
+     There is a single API endpoint: `/mecplatforms?filter=closest`. To call this endpoint, the API consumer must first obtain a valid OAuth2 token from the token endpoint, and pass it as an `Authorization` header in the API request.
+
+     
+     The API returns the closest MEC platform to a given device, so that device needs to be identifiable by the network:
+    * if you call the API from a server, you must explicitly pass one or more device identifiers in the HTTP request header:
+      * `IP-Address`. This is the public IP address of the user device: this can be obtained by an application hosted on that device calling a public IP address API (e.g. GET https://api.ipify.org?format=json)
+      * `Phone-Number` . The international E.164 format (starting with country code), e.g. +4407123123456
+      * `Network-Access-Identifier` (where available from the API host operator)
+      
+    * if you call the API from a device attached to the operator network, you _may_ omit the explicit device identifier(s)from the request header. If such a request fails with a `404 Not Found` error then retry the request but this time include a device identifier.
+
+
+    
+    # Example requests:
+    
+    Examples for all API clients:
+    
+        GET /mec-platforms?filter=closest HTTP/1.1
+        Host: example.com
+        Accept: application/json
+        Network-Access-Identifier: 4d596ac1-7822-4927-a3c5-d72e1f922c94@domain.com
+
+        GET /mec-platforms?filter=closest HTTP/1.1
+        Host: example.com
+        Accept: application/json
+        IP-Address: 84.125.93.10
+        
+        GET /mec-platforms?filter=closest HTTP/1.1
+        Host: example.com
+        Accept: application/json
+        Phone-Number: 441234567890
+        
+    Example where API client is on a network-attached device:
+    
+        GET /mec-platforms?filter=closest HTTP/1.1
+        Host: example.com
+        Accept: application/json
+ 
+    # Responses
+    
+    ## Success
+    
+    A JSON object is returned containing a `MECPlatforms` array with a single member, along with the HTTP `200 OK` status code. The value of the Ern (Edge Resource Name) object is the name of the closest MEC platform to the user device. An example of this JSON object is as follows:
+    ```
+    {
+      "MecPlatforms": [
+        {
+          "Provider": "AWS",
+          "Ern": "eu-west-2-wl1-man-wlz-1
```

Comment:
Missing closing " at the end of eu-west-2-wl1-man-wlz-1

Reply:
Thanks @RandyLevensalor , fixed with latest commit